docs/sources/panels/visualizations/geomap.md

```diff
@@ -97,26 +97,26 @@ The Geomap panel needs a source of geographical data. This data comes from a dat
 - **Geohash** specifies that your query holds geohash data. You will get prompted to select a string data field for the geohash from your database query.
 - **Lookup** specifies that your query holds location name data that needs to be mapped to a value. You will get prompted to select the lookup field from your database query and a gazetteer. The gazetteer is the directory that is used to map your queried data to a geographical point.
 
-### Markers Layer
+### Markers layer
 
 The markers layer allows you to display data points as different marker shapes such as circle, squares, triangles, stars, and more.
 
 ![Markers Layer](/static/img/docs/geomap-panel/geomap-markers-8-1-0.png)
 
 ![Markers Layer Options](/static/img/docs/geomap-panel/geomap-markers-options-8-1-0.png)
 
-- **Marker Color** configures the color of the marker. The default `Fixed size` keeps all points a single color. There is an alternate option to have multiple colors depending on the data point values and the threshold set at the `Thresholds` section.
-- **Marker Size** configures the size of the marker. Default is `Fixed size`, making all marker size the same regardless of the data points. However, there is also an option to scale the circles to the corresponding data points. `Min` and `Max` marker size has to be set such that the Marker layer can scale within these range.
-- **Marker Shape** provides you with the flexibility to visualize the data points differently. 
+- **Marker color** configures the color of the marker. The default `Fixed size` keeps all points a single color. There is an alternate option to have multiple colors depending on the data point values and the threshold set at the `Thresholds` section.
+- **Marker size** configures the size of the marker. Default is `Fixed size`, making all marker size the same regardless of the data points. However, there is also an option to scale the circles to the corresponding data points. `Min` and `Max` marker size has to be set such that the Marker layer can scale within these range.
+- **Marker shape** provides you with the flexibility to visualize the data points differently. 
```

Comment:
What does the UI say? We should stay true to the UI for labels. Thanks :)

Reply:
Makes sense, those are changed now. I wasn't sure if I should use sentence case for these as well. :)

Reply:
No worries! Many thanks for your contribution!! I will upload the images, then do a final review and merge.